be/src/filesystem/local_file_system.cpp

```diff
@@ -0,0 +1,126 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "filesystem/local_file_system.h"
+
+#include <fmt/format.h>
+
+#include <filesystem>
+#include <system_error>
+
+#include "filesystem/read_stream.h"
+#include "filesystem/write_stream.h"
+
+namespace fs = std::filesystem;
+
+namespace doris {
+
+LocalFileSystem::LocalFileSystem(std::string root_path) : _root_path(std::move(root_path)) {}
+
+LocalFileSystem::~LocalFileSystem() = default;
+
+Status LocalFileSystem::exists(const std::string& path, bool* res) const {
```

Comment:
This class is similar to PosixEnv, most functions of it is also defined in PosixEnv, is it necessary to create a new class.